File: ClientApp/src/app/_models/therapiarecorddto.ts
export interface TherapiaRecordDto {
    id: number;
    medicalRecordId: number;
    therapiaId: number;
    therapiaName: string;
    therapiaUnit: string;
    amount: number;
}

export interface LocalTherapiaRecord {
    id: number,
    dbId?: number
    therapiaName: string,
    therapiaId: number,
    amount: number
}

export interface TherapiaOnMedicalRecord {
    therapiaId: number;
    amount: number;
}

File: Models/DTOs/MedicalRecord/TherapiaRecordDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Vet.Models.DTOs.MedicalRecord
{
    public class TherapiaRecordDto
    {
        public int Id { get; set; }

        public int MedicalRecordId { get; set; }

        public int TherapiaId { get; set; }
        public string TherapiaName { get; set; }
        public string TherapiaUnit { get; set; }

        public double Amount { get; set; }
    }
}
